### ISSUE TITLE
Handle network request failed in react-native gracefully

### ISSUE THREAD

**Issue by SailingSteve** [wevote/WeVoteReactNative#62]
<img width="1588" alt="screen shot 2017-09-13 at 4 45 47 pm" src="https://user-images.githubusercontent.com/6923653/30405881-5c109ff6-98a3-11e7-8e15-793834ef2e12.png">

**Comment by RohanVB** [comment]
Unable to reproduce when launching the app from terminal and using the remote debugger/dev tools

![screen shot 2017-10-12 at 11 07 58 pm](https://user-images.githubusercontent.com/15197846/31510467-504faf8e-afa2-11e7-9b75-19711f30a76f.png)

**Comment by DaleMcGrew** [comment]
Hi @SailingSteve, is this issue still active?

**Comment by SailingSteve** [comment]
Yes still active, but there may no longer be a red exception.  It would be nice to notify the user that the app can't contact the server, either due to device settings, or a downed site.
